include/swift/AST/ModuleDependencies.h

```diff
@@ -767,6 +768,9 @@ class SwiftDependencyScanningService {
     ModuleDependenciesKindMap ModuleDependenciesMap;
   };
 
+  /// The CASOption created the Scanning Service if used.
+  Optional<clang::CASOptions> CASOpts;
```

Comment:
Why not `llvm::Optional`?

Reply:
This is apparently a mistake. I was intended to update this Optional to llvm::Optional but I accidentally modify the one above.